tape/src/main/java/com/squareup/tape/FileException.java

```diff
@@ -6,6 +6,8 @@
 
 /** Encapsulates an {@link IOException} in an extension of {@link RuntimeException}. */
 public class FileException extends RuntimeException {
+  private static final long serialVersionUID = 1L;
```

Comment:
Non-gradle changes start here (lots of Java 11 changes) then the actual payload.

Reply:
Why is this `serialVersionUID` field needed here, what's changed?